misc/cron/.htaccess

```diff
@@ -1,5 +1,15 @@
 # Allow direct web access to Web cron
-<Files "archive.php">
-	Order Allow,Deny
-	Allow from all
-</Files>
+# Apache 2.2
+<IfModule !authz_core_module>
```

Comment:
Looks like we already do something similar in some other places, except instead of `<IfModule !authz_core_module>`, we use `<IfVersion`: https://github.com/matomo-org/matomo/blob/2b35b46a0988f8c3b6f848c59be8d7d918965a52/plugins/Installation/ServerFilesGenerator.php#L251-L259

Could you change this to use that syntax to keep things consistent?